fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/impl/networking/UntrackedNetworkHandler.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.networking;
+
+// An internal marker interface used by the fake player API to prevent the network addon from being tracked.
+public interface UntrackedNetworkHandler {
```

Comment:
should this be `@ApiStatus.Internal`?

Reply:
No, this annotation is automatically applied to all impl packages, meaning as its in the impl package it will effectivity already have this annotation. See: https://github.com/FabricMC/fabric/blob/1.21/gradle/package-info.gradle